btas/generic/swap.h

```diff
@@ -0,0 +1,102 @@
+#ifndef BTAS_SWAP
+#define BTAS_SWAP
+#ifdef _HAS_INTEL_MKL
+#include <mkl_trans.h>
+#include <btas/btas.h>
+#include <vector>
+					//***IMPORTANT***//
+//do not use swap to first then use swap to back
+//swap to first preserves order while swap to back does not
+//If you use swap to first, to undo the transpositions
+//make LD_in_front = true and same goes for swap to back
+//do not mix swap to first and swap to back
+namespace btas{
+	/// Swaps the nth mode of an Nth order tensor to the front preserving the 
```

Comment:
seems like you used tabs here! tabs are a big no-no.

I will push a .clang-format into BTAS master, pull it into your branch and reformat. This will fix this issue.